js/modules/data.js

```diff
@@ -2,18 +2,21 @@
 
 const electron = require('electron');
 
+// TODO: this results in poor readability, would be
+// much better to implicitly call with `_`.
```

Comment:
Did you mean "explicitly" instead of implicitly ?

Reply:
Yes :)

Reply:
updated